plugin-modernizer-core/src/main/java/io/jenkins/tools/pluginmodernizer/core/config/Settings.java

```diff
@@ -32,6 +32,8 @@ public class Settings {
 
     public static final String RECIPE_DATA_YAML_PATH = "recipe_data.yaml";
 
+    public static final String UPDATE_CENTER_URL = "https://updates.jenkins.io/current/update-center.actual.json";
```

Comment:
Shouldn't that URL be out of the code, in a properties file or something that would allow us to change it without rebuilding?

Reply:
Yea like the Plugin installation manager tool this must be configurable